src/Cake/Program.cs

```diff
@@ -68,8 +69,9 @@ public static int Main()
                     builder.Update(container);
 
                     // Load all modules.
-                    var loader = container.Resolve<ModuleLoader>();
-                    loader.LoadModules(container, options);
+                    var moduleSearcher = container.Resolve<ModuleSearcher>();
+                    var moduleLoader = container.Resolve<ModuleLoader>();
+                    moduleLoader.LoadModules(moduleSearcher.Search(options), container, container.Resolve<ScriptConfiguration>());
```

Comment:
What's the reason for this change (searching for modules outside of the ModuleLoader)?